base/bitarray.jl

```diff
@@ -236,7 +236,9 @@ function fill!(B::BitArray, x)
 end
 
 falses(args...) = fill!(BitArray(args...), false)
```

Comment:
I think we should also restrict the signature here. As this PR shows, too broad signatures can give unexpected results. Looks like we need two methods: `falses(::Dims)` and `falses(::Integer...)`.